go.mod

```diff
@@ -1,102 +1,89 @@
 module github.com/pressly/goose/v3
 
-go 1.20
+go 1.21
+
+toolchain go1.22.1
 
 require (
-	github.com/ClickHouse/clickhouse-go/v2 v2.17.1
-	github.com/go-sql-driver/mysql v1.7.1
-	github.com/jackc/pgx/v5 v5.5.2
-	github.com/marcboeker/go-duckdb v1.5.6
+	github.com/ClickHouse/clickhouse-go/v2 v2.20.0
+	github.com/go-sql-driver/mysql v1.8.0
+	github.com/jackc/pgx/v5 v5.5.5
+	github.com/marcboeker/go-duckdb v1.6.1
 	github.com/mfridman/interpolate v0.0.2
-	github.com/microsoft/go-mssqldb v1.6.0
-	github.com/ory/dockertest/v3 v3.10.0
```

Comment:
I want the main goose module to have no dependency on this and any of its deps. I think removing it and having an independent "integration test" module is cleaner.